crowbar_framework/app/models/node_object.rb

```diff
@@ -1249,6 +1249,22 @@ def poweroff
     bmc_cmd("power off")
   end
 
+  def service(service_name, action)
+    case service_name
+    when :chef
+      provider_service_object = ChefObject.get_provider_service_instance(self, "chef-client")
+    end
+
+    case action
+    when :start
+      provider_service_object.start_service
```

Comment:
That will never work. This command gets executed on the admin node. You need to execute it on the node itself.